controllers/lvmcluster_controller.go

```diff
@@ -208,6 +215,46 @@ func (r *LVMClusterReconciler) reconcile(ctx context.Context, instance *lvmv1alp
 	return ctrl.Result{}, nil
 }
 
+func (r *LVMClusterReconciler) verifyLvmClusterSpec(ctx context.Context, instance *lvmv1alpha1.LVMCluster) error {
+
+	// make sure no device overlap with another VGs
```

Comment:
what if the user decides to give the device names instead of device paths?
I know we encourage them to use paths, but still if they decide to use device names instead, then this validation will always fail, if I'm not wrong.

Reply:
Even in that case, it should work, as the device names will only be the same. I am just loading the devices into a map; if the key already exists, we are erroring out.